docs/BasicCommunication/GetFilePath/index.md

```diff
@@ -0,0 +1,81 @@
+## GetFilePath
+
+Type
+: Request
+
+Sender
+: SDL
+
+Purpose
+: Retrieve a file path from the HMI app service provider
+
+### Request
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|fileName|String|true|maxlength: 255|
+|fileType|[Common.FileType](../../common/enums/#filetype)|false||
+|appServiceId|String|false||
+
+
+### Response
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|filePath|String|false||
+|fileType|[Common.FileType](../../common/enums/#filetype)|false||
+
+### Sequence Diagrams
+|||
+GetFile (HMI Provider)
```

Comment:
The diagram does show the general `GetFile` flow so this title is more specific

Reply:
I was going to say that it was inconsistent with the rest of the guidelines, but I'm less certain now after looking. Still a bit odd, since `GetFile` is a Mobile RPC, not an HMI RPC, but I suppose it's not that important.